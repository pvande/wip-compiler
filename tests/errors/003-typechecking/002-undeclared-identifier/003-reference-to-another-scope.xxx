x := () => {
  a := "inner scope"
}

y := a
