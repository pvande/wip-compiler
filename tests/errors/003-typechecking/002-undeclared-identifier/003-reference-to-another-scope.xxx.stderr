

Error: "Could not infer the type of this variable"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/003-reference-to-another-scope.xxx[0m on line [1;37m5[0m

> [0;36my := a[0m
  [0;31m^     [0m
Error: "Declaration for that identifier was not found"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/003-reference-to-another-scope.xxx[0m on line [1;37m5[0m

> [0;36my := a[0m
  [0;31m     ^[0m
